src/ResourceManager/Storage/Commands.Management.Storage/Commands.Management.Storage.csproj

```diff
@@ -67,10 +67,13 @@
       <HintPath>..\..\..\packages\Microsoft.Data.Services.Client.5.8.2\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
```

Comment:
@blueww you can remove the references added in this file (`Microsoft.Rest.*`, `Newtonsoft.Json`) since they are already referenced by the `Common.Dependencies.targets` file